drawBot/context/baseContext.py

```diff
@@ -3013,7 +3025,10 @@ def _getNSFontFromNameOrPath(fontNameOrPath, fontSize, fontNumber):
 
 @memoize
 def getFontDescriptorsFromPath(fontPath):
-    modTime = os.stat(fontPath).st_mtime
+    if os.path.exists(fontPath):
+        modTime = os.stat(fontPath).st_mtime
+    else:
+        modTime = -1
     prevModTime, descriptors = _reloadedFontDescriptors.get(fontPath, (modTime, None))
```

Comment:
```suggestion
    prevModTime, descriptors = _reloadedFontDescriptors.get(fontPath, (modTime, ())
```
This allows you to remove the `if descriptors is None` test below.

Reply:
changed

Reply:
`tuple()` is redundant, though, plain `()` means exactly the same.